docs/recipes/htmlhint.md

```diff
@@ -30,25 +30,48 @@ Read [gulp-htmlhint's docs][api] to find out how you can pass options to HTMLHin
 
 HTMLHint should run on serve and build.
 
-```js
-gulp.task('html', ['htmlhint', 'styles', 'scripts'], () => {
+```diff
```

Comment:
I moved to diff syntax highlight since I find it more readable.

It's not aligned with what the issue #591 is mentioning but please let me know if you find this acceptable.

Thanks!

Reply:
Here it looks great 👌 I disliked it in recipes with larger changes 😄